cmd/utils/flags.go

```diff
@@ -659,6 +661,16 @@ func MakeSystemNode(name, version string, extra []byte, ctx *cli.Context) *node.
 	// Configure the Ethereum service
 	accman := MakeAccountManager(ctx)
 
+	// initialise new random number generator
+	rand := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
The only source of entropy here is time. Whether you get the JitVM or not depends entirely on the time when you start geth. I don't know whether that's an issue or not, just feels weird.